### ISSUE TITLE
Web components: Modal - keyboard and screen reader issues on Expressive Modal

### ISSUE THREAD

**Issue by Praveen-sr** [carbon-design-system/ibm-dotcom-library#3162]
## Detailed description

Issue :- 
1) Close icon
2) 'Lorem ipsum dolor'  button 
Mentioned elements are not keyboard operable.

Expected :
Each and every interactive element should be keyboard operable.

**Comment by asudoh** [comment]
#3134 addresses the issue with the close button. The primary button is a dummy button and should do nothing, but if you hit Space key on the primary button you'll see there is a UI feedback that it's hit (in the same manner as when you click it by mouse).
